manifests/deis-router-rc.yaml

```diff
@@ -23,3 +23,7 @@ spec:
         ports:
         - containerPort: 80
           hostPort: 80
+        - containerPort: 443
+          hostPort: 443
+        - containerPort: 2222
```

Comment:
Could builder do that host:2222 -> container:2222 port mapping instead?  Yep... but then there'd be two points of ingress into Deis (2222 and 80/443), which would necessitate _two_ load balancers external to the cluster and some fancier DNS records that direct builder traffic to the second LB.  Bottom line here is the strategy I'm employing is actually identical to the one used in 1.x.